managegroup.php

```diff
@@ -257,185 +263,141 @@
-		}
-		else
-		{
-			$email = '';
-		}
-
-		$user['username'] = htmlspecialchars_uni($user['username']);
-		$user['username'] = format_name($user['username'], $user['usergroup'], $user['displaygroup']);
-		$user['profilelink'] = build_profile_link($user['username'], $user['uid']);
-		if(in_array($user['uid'], $leaders_array))
-		{
-			$leader = $lang->leader;
-		}
-		else
-		{
-			$leader = '';
-		}
-
-		// Checkbox for user management - only if current user is allowed
-		$checkbox = '';
-		if($groupleader['canmanagemembers'] == 1)
-		{
-			eval("\$checkbox = \"".$templates->get("managegroup_user_checkbox")."\";");
-		}
-
-		eval("\$users .= \"".$templates->get("managegroup_user")."\";");
-	}
-
-	if(!$users)
-	{
-		eval("\$users = \"".$templates->get("managegroup_no_users")."\";");
-	}
-
-	$add_user = '';
-	$remove_users = '';
-	if($groupleader['canmanagemembers'] == 1)
-	{
-		eval("\$add_user = \"".$templates->get("managegroup_adduser")."\";");
-		eval("\$remove_users = \"".$templates->get("managegroup_removeusers")."\";");
-	}
-
-	if($usergroup['type'] == 5 && $groupleader['caninvitemembers'] == 1)
-	{
-		eval("\$invite_user = \"".$templates->get("managegroup_inviteuser")."\";");
-	}
-
-	$plugins->run_hooks("managegroup_end");
-
-	eval("\$manageusers = \"".$templates->get("managegroup")."\";");
-	output_page($manageusers);
+    $plugins->run_hooks("managegroup_start");
+
+    $lang->members_of = $lang->sprintf($lang->members_of, htmlspecialchars_uni($usergroup['title']));
+    $lang->add_member = $lang->sprintf($lang->add_member, htmlspecialchars_uni($usergroup['title']));
+    $lang->invite_member = $lang->sprintf($lang->invite_member, htmlspecialchars_uni($usergroup['title']));
+    $usergroup['pendingrequests'] = false;
+
+    if ($usergroup['type'] == 5) {
+        $usergroup['usergrouptype'] = $lang->group_public_invite;
+    }
+    elseif ($usergroup['type'] == 4) {
+        $query = $db->simple_select("joinrequests", "COUNT(*) AS req", "gid='{$gid}'");
+        $numrequests = $db->fetch_array($query);
+
+        if ($numrequests['req']) {
+            $usergroup['pendingrequests'] = true;
+            $lang->num_requests_pending = $lang->sprintf($lang->num_requests_pending, $numrequests['req']);
+        }
+        $usergroup['usergrouptype'] = $lang->group_public_moderated;
+    }
+    elseif ($usergroup['type'] == 3) {
+        $usergroup['usergrouptype'] = $lang->group_public_not_moderated;
+    }
+    elseif ($usergroup['type'] == 2) {
+        $usergroup['usergrouptype'] = $lang->group_private;
+    } else {
+        $usergroup['usergrouptype'] = $lang->group_default;
+    }
+
+    $usergroup['leaders'] = false;
+
+    // Display group leaders (if there is any)
+    $query = $db->query("
+        SELECT g.*, u.username, u.usergroup, u.displaygroup
+        FROM " . TABLE_PREFIX . "groupleaders g
+        LEFT JOIN " . TABLE_PREFIX . "users u ON (u.uid=g.uid)
+        WHERE g.gid = '{$gid}'
+    ");
+
+    $leaders_array = array();
```

Comment:
For consistency with the below `$leaders = []`, can you use square brackets here too to initialise the array?